src/core_plugins/kibana/public/visualize/listing/create_or_delete_button.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import { CreateButtonLink } from 'ui_framework/components/button/create_button_link';
+import { DeleteButton } from 'ui_framework/components/button/delete_button';
+
+export function CreateOrDeleteButton({ showCreate, doDelete }) {
+  if (showCreate) {
+    return <CreateButtonLink href = "#/visualize/new"
+                             tooltip = "Create new visualization" />;
```

Comment:
Not a fan of this kind of aligning. What about this?

```
return (
  <CreateButtonLink
    href = "#/visualize/new"
    tooltip = "Create new visualization" />
);
```

(This works nicely with deeper structures, for examples. It also doesn't break on "search and replace" and things like that. Hopefully [prettier](https://github.com/jlongster/prettier) becomes production-ready really fast, so we no longer have to think about these kinds of things 🎉)